packages/react-dev-overlay/src/middleware.ts

```diff
@@ -225,7 +226,7 @@ export async function createOriginalStackFrame({
       ? path.relative(rootDirectory, filePath)
       : sourcePosition.source,
     lineNumber: sourcePosition.line,
-    column: sourcePosition.column,
+    column: (sourcePosition.column ?? 0) + 1,
```

Comment:
Why are we doing that? Could we just make it 0-indexed instead

Reply:
We could create our own stack frame that is 0-based and create them from the 1-based frames that `parseStack` returns, but I think that may be even more confusing.